### DISCUSSION TITLE
TuneIn (Premium) + Beta

### DISCUSSION THREAD

**Discussion by chicknlil** [music-assistant/support#1190] [Feature requests and ideas]
Hi all,

New to Music Assistant (saw it rec'd on Reddit and was intrigued, so I jumped in. Spotify set up with minimal hiccups. But I'm having some issues with TuneIn (I have a premium account), and nothing in troubleshooting seems applicable (there also doesn't seem to be specific documentation instructions for TuneIn).

The setup for TuneIn only asks for my username (not my password - which makes me wonder if it's even connecting) and it doesn't pull in the [one radio I listen to](https://tunein.com/radio/Boston-Bruins-s137387/). When I started playing with this addon, the station was broadcasting (as the Bruins were playing) but the game has since ended, and the station is offline, which could explain why I'm not seeing anything now, but doesn't really explain why I wouldn't have in the first place.

Any thoughts, suggestions? Being able to use HA to play music + the hockey games to my various speakers (all of which were detected with no issues) will be a far better solution versus what I'm using currently.

Aside - I did see a discussion relating to TuneIn premium, but I don't believe these issues are related, so I didn't want to piggyback on that thread.

Much obliged for anyones help!

**Comment by OzGav** [comment]
MA shows you the stations you have in your favourites https://tunein.com/user/YOUR_USER_NAME/favorites

**Comment by chicknlil** [comment]
![image](https://user-images.githubusercontent.com/100124187/231331357-08e92d26-f7da-4fe2-94da-1f4899c1b6ba.png)

So it's there when I modify that path to my username.

But MA shows:

![image](https://user-images.githubusercontent.com/100124187/231331524-78cecf3d-341f-471c-96ba-c6b514cf40dc.png)

**Reply by OzGav** [reply]
Hmmmm....Maybe because it is unavailable but normally you get a message about that. You can try the overflow menu top right and resync. Also try adding another station from some other station that is well used and see if that appears.

**Reply by chicknlil** [reply]
That worked, though I had to navigate away from the Radio section before it populated (using the overflow menu didn't trigger it, oddly).

My team plays again Thursday night at 7 PM ET.  Will try again at that point to see if their radio network is available.

So, if you wouldn't mind, I'd like to keep this open until that point.

Thank you for your quick responses and assistance thus far!

**Reply by OzGav** [reply]
After the resync you should see a red dot next to the reload icon (second from left) 
![image](https://user-images.githubusercontent.com/19848947/231333365-4ef295cb-addc-4c24-802f-6c1a5dedae71.png)

Pressing that either way should refresh the list

**Comment by chicknlil** [comment]
Game is live now:
![image](https://user-images.githubusercontent.com/100124187/231903079-e632a603-156a-46eb-b10a-811936f73160.png)

But is still not showing among available radios:
![image](https://user-images.githubusercontent.com/100124187/231903139-89adb8ea-dfbe-4e07-9c34-c1c3aaf2126f.png)

I have tried triggering a refresh via the overflow menu, hitting the resyc button, navigating away from the radion section, and even refreshing the entire browser and clearing the cache. No joy.

**Reply by OzGav** [reply]
If other stations that you have added to your library are appearing then post the url for the station here and we can see if @marcelveldt can look into it at some point.

**Reply by chicknlil** [reply]
Sure, the Bruins radio network link was in my original post, as well - https://tunein.com/radio/Boston-Bruins-s137387/.

Could it be something to do with it being a premium-only station?

**Reply by OzGav** [reply]
Yes it absolutely could be and probably is but I will confirm

**Reply by chicknlil** [reply]
> Yes it absolutely could be and probably is but I will confirm

Hello @OzGav ! Just wanted to circle back and check for an update.  I imagine this probably pretty far down the list of things to do (as TuneIn, in general, works) but...yeah. :)

**Reply by OzGav** [reply]
Have discussed with the team and we are changing this to a feature request. At this time we don’t have capacity to set up and test TuneIn premium. Sorry we can’t be more responsive in this instance but hope to be able to support this in the future.

**Reply by chicknlil** [reply]
Understood. My team unceremoniously exited the playoffs last night, so I'd not be looking for it again until sometime in October.

Thanks for your time.
